pkg/rtc/transport.go

```diff
@@ -1440,6 +1432,11 @@ func (t *PCTransport) filterCandidates(sd webrtc.SessionDescription, preferTCP,
 					continue
 				}
 				excluded := preferTCP && !c.NetworkType().IsTCP()
+				if !excluded {
+					if !t.params.Config.UseMDNS && types.IsICECandidateMDNS(c) {
+						excluded = true
+					}
```

Comment:
This will exclude the mDNS candidates sent into Pion. Previously, only trickled mDNS candidates were filtered. mDNS candidates as part of the full SDP did not get filtered.

Reply:
nice!